pymc3/examples/ATMCMC_2gaussians.py

```diff
@@ -0,0 +1,68 @@
+import pymc3 as pm
+import numpy as np
+from pymc3.step_methods import atmcmc
+import theano.tensor as tt
+from matplotlib import pyplot as plt
+
+test_folder = ('ATMIP_TEST')
+
+n_chains = 300
+n_steps = 100
+tune_interval = 25
+n_jobs = 1
+
+n = 4
+
+mu1 = np.ones(n) * (1. / 2)
+mu2 = -mu1
+
+stdev = 0.1
+sigma = np.power(stdev, 2) * np.eye(n)
+isigma = np.linalg.inv(sigma)
+dsigma = np.linalg.det(sigma)
+
+w1 = stdev
+w2 = (1 - stdev)
+
+
+def last_sample(x):
+    return x[(n_steps - 1)::n_steps]
+
+
+def two_gaussians(x):
```

Comment:
I tried it just now, but I cannot get it running. Could please someone update the help of the shape of the arrays that are expected? My two mu1 and mu2 are shape (1 x n) how is the total mu matrix expected to be? 2 x n? 1 x 2*n? How are the weights and taus expected?